scripts/generate-mod.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
@feloy Unless I've missed something about the usage of this script, I'm wondering if it wouldn't make sense to run this as part of CI, to detect cases where the `go.mod` file would be edited directly without updating this script (e.g, Dependabot PRs or someone updating/adding a new dependency). This way, we would be aware of editing this file if there is a diff.
But I guess it might not be straightforward especially where this script might potentially update dependencies with `go get -u`..

Reply:
I agree that I would make sense. I remember we discussd to put in place a script to check these kind of generated code (mocks at least); that would be part of this work

Reply:
Indeed, I also remember a discussion around such checks. I'll try to find a related issue (or create a new one) to track this.